src/components/common/Balance.js

```diff
@@ -8,24 +8,35 @@ const CustomDiv = styled(List)`
     display: inline;
 `
 
+const FRAC_DIGITS = 5
+
 const Balance = ({ amount }) => {
     if (!amount) {
         throw new Error("amount property should not be null")
     }
     let amountShow = formatNEAR(amount)
+
     return (
-        <CustomDiv>
+        <CustomDiv title={['0','0.0001'].includes(utils.format.formatNearAmount(amount, (FRAC_DIGITS || 1) - 1)) ? `${formatWithCommas(amount)} yoctoⓃ` : ``}>
             {amountShow} Ⓝ
         </CustomDiv>
     )
 }
 
 export const formatNEAR = (amount) => {
-    let ret =  utils.format.formatNearAmount(amount, 5);
+    let ret =  utils.format.formatNearAmount(amount, FRAC_DIGITS);
     if (ret === '0') {
-        return "<0.00001";
+        return `<${!FRAC_DIGITS ? `0` : `0.${'0'.repeat((FRAC_DIGITS || 1) - 1)}1`}`
     }
     return ret;
 }
 
+const formatWithCommas = (value) => {
```

Comment:
JavaScript has [the `Intl.NumberFormat` utility](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat) built in, which will allow us to display large numbers in the format users are most used to, rather than always using commas. It can be used like this:

```js
new Intl.NumberFormat().format(amount)
```

I'd prefer we use this! 

Note that passing no arguments to `NumberFormat()` results in it selecting the user's locale. See [the docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat) for info about how to try it with other locale settings.

Reply:
@chadoh @marcinbodnar  it doesn't work because we have more than 53-bit numbers. Generally we cannot use built-in JS number support for anything related to NEAR amounts (or any other number that can have more than 53 bits of precision required).

Reply:
Could we use a library like [bignumber.js](https://github.com/MikeMcl/bignumber.js/) rather than implementing this logic ourselves?

Reply:
@chadoh we already bring https://github.com/indutny/bn.js/ for big number math, ideally we shouldn't be bringing 2 libraries (as it adds weight).  It might be reasonable to replace `BN` with `bignumber.js`,  but it would require a lot of changes around `nearlib`, `near-shell` and app examples. Given that we are trying to stabilize for release, I think it's best to avoid doing it pre-mainnet.

Reply:
I've enjoyed using bignumber.js more than BN, but I agree that we should not add a second library or do refactoring now. Looks like BN does not support formatting with commas or other separators, so we will need to implement some sort of `formatWithCommas` ourselves for now.

Reply:
@chadoh I agree with your sentiment on bignumber.js vs BN, just think that changing that globally is pretty big project by itself, so we shouldn't block this change.